modules/ollama/src/main/java/org/testcontainers/ollama/OllamaContainer.java

```diff
@@ -23,6 +23,7 @@
 public class OllamaContainer extends GenericContainer<OllamaContainer> {
 
     private static final DockerImageName DOCKER_IMAGE_NAME = DockerImageName.parse("ollama/ollama");
+    private static final int OLLAMA_PORT = 11434;
```

Comment:
```suggestion

    private static final int OLLAMA_PORT = 11434;
```

Reply:
I'd revisit this later for all modules. Let's keep it private for now, for consistency.